### PR TITLE
create file structure for client side relates #1

### PR DESCRIPTION
**Pull request by shathakh**

### CODE REVIEW

**Review by naghamabuwarda — APPROVED** [review]

**Review by alishanti98 — APPROVED** [review]

**Review by mohammedagha27 — APPROVED** [review]

File: client/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/recommended"
```
Comment (LinaYahya) [inline client/.eslintrc.json:10]: add "plugin:react/jsx-runtime"  to extend so I dont have to import react when writing jsx

File: client/public/index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (LinaYahya) [inline client/public/index.html:6]: add description meta data

File: client/.eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": "latest"
+    },
+    "plugins": [
+        "react",
+        "@typescript-eslint"
+    ],
+    "rules": {
+    }
```
Comment (LinaYahya) [inline client/.eslintrc.json:23]:
```suggestion
    "linebreak-style": 0,
      "max-len": ["error", { "code": 80 }]
    }
```

**Review by LinaYahya — COMMENTED** [review]
I think airbnb style guide wasn't chosen when installing eslint, So we're going to install eslint again
Also, let's us create a basic style for our app, container margin, colors, font, and container style on large screens

**Review by LinaYahya — APPROVED** [review]

**Pull request merged**